### PR TITLE
Add May 2021 community call notes

### PR DESCRIPTION
**Pull request by isabela-pf**
May's community call had a lot of surprises. Here are the notes.

Thanks again to whomever reviews this!

### CODE REVIEW

File: docs/source/community/community-call-notes/2021-may.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+* **Darian** Congratulations to the [2020 cohort of Jupyter Distinguished Contributors](https://blog.jupyter.org/congratulations-distinguished-contributors-149deff3033c)!
+* **Nick** [ipydrawio 1.1.0](https://github.com/deathbeds/ipydrawio/releases/tag/v1.1.0) _just_ dropped: now has the `sketch` theme, a _Custom Diagram..._ Launcher which offers all of the (extensible) templates. Also [docs](https://ipydrawio.rtfd.io).
+* **Steve** [JupyterLab 3.1.0a9](https://github.com/jupyterlab/jupyterlab/releases/tag/v3.1.0a9) is now available with Real Time Collaboration (RTC)!  To use:
+    * Install
+      * `pip install -U --pre jupyterlab` or
+      * `conda install -c conda-forge -c conda-forge/label/jupyterlab_alpha jupyterlab=3.1.0a9`
+    * Usage 
+      * `jupyter lab --collaborative` or
+      * Jupyter settings: `{"LabApp": {"collaborative": true, "expose_app_in_browser": true}}`
+* **Isabela** Congratulations to all the community members who sumbitted CZI grant proposals last week! Thank you for your hard work.
```
Comment (bl-ue) [inline docs/source/community/community-call-notes/2021-may.md:31]:
```suggestion
* **Isabela** Congratulations to all the community members who submitted CZI grant proposals last week! Thank you for your hard work.
```

**Review by bl-ue — APPROVED** [review]
One typo.

File: docs/source/community/community-call-notes/2021-may.md (hunk 1 above)
Comment (isabela-pf) [reply docs/source/community/community-call-notes/2021-may.md:31]: Thanks for catching it!

**Comment by bl-ue** [comment]
@isabela-pf on a different note, I fixed typos in the older community call notes files, in #558.  Is that fine, or are those pages supposed to be archives never modified again?

**Comment by isabela-pf** [comment]
@bl-ue that's great, thanks! I don't think there are any rules about how to handle those notes long-term besides making sure they remain openly available.

**Pull request merged**